### ISSUE TITLE
BST Sequence

### ISSUE THREAD

**Issue by sanazkh** [maiquynhtruong/algorithms-and-problems#1]
The BST sequence code does not work properly. It adds the already chosen nodes to the list.

**Comment by maiquynhtruong** [comment]
Hey, thanks for checking out my code. I really appreciate it! 😃 

While Hacktoberfest is going on, do you want to fix that and make a PR? I can add the Hacktoberfest label to this issue.

If not, I'll try to fix it ASAP.

Thanks again 😄

**Comment by maiquynhtruong** [comment]
Hey did you take Codepath's interview prep this summer too? 😮

**Comment by sanazkh** [comment]
Yes! were you there as well? :D

**Comment by maiquynhtruong** [comment]
Yeah. I think we did a video mock interview once. I still remember your face and you went to SJSU haha

**Comment by maiquynhtruong** [comment]
But you if want, you can definitely create a PR for this. Thanks for checking my repo out 😆
